lib/js/src/manager/SystemCapabilityManager.js

```diff
@@ -749,6 +750,18 @@ class SystemCapabilityManager extends _SubManagerBase {
             && defaultMainWindow.getImageTypeSupported() !== undefined
             && defaultMainWindow.getImageTypeSupported().length > 0);
 
+        const displayCapabilitiesOld = this.getDisplayCapabilities();
+
+        if (displayCapabilitiesOld !== null && displayCapabilitiesOld !== undefined) { // The capability exists
+            // Copied from the RAI response, since this parameter is not present in WindowCapability
+            convertedCapabilities.setScreenParams(displayCapabilitiesOld.getScreenParams());
+            if (displayCapabilitiesOld.getMediaClockFormats() === null) {
+                convertedCapabilities.setMediaClockFormats([]); // insert empty array since this parameter is mandatory
```

Comment:
You are still setting MediaClockFormats above to an empty array, So I don't think you need to do it down here, you just need to check something like:

```
if (displayCapabilitiesOld.getMediaClockFormats() != null) {
convertedCapabilities.setMediaClockFormats(displayCapabilitiesOld.getMediaClockFormats());
}
```

What you have does work tho, let me know what you think.